config/src/config/storage_config.rs

```diff
@@ -143,7 +143,7 @@ impl Default for LedgerPrunerConfig {
     fn default() -> Self {
         LedgerPrunerConfig {
             enable: true,
-            prune_window: 10_000_000,
+            prune_window: 150_000_000,
```

Comment:
nit: Can you add a comment explaining the reasoning/calculation behind these numbers?

Reply:
+1 I'd like a rationale for this number commented in.

Reply:
Comment added. (tl;dr it's still kind of magic numbers.)